docs/api/qiskit/0.44/release-notes/0.40.md

```diff
@@ -21,39 +21,39 @@ This release officially deprecates the Qiskit IBMQ provider project as part of t
 
 Qiskit Terra 0.23.0 is a major feature release that includes a multitude of new features and bugfixes. The highlights for this release are:
 
-> *   Support for importing OpenQASM 3 programs and creating [`QuantumCircuit`](qiskit.circuit.QuantumCircuit "qiskit.circuit.QuantumCircuit") objects from the input program via two new functions [`qiskit.qasm3.load()`](qasm3#qiskit.qasm3.load "qiskit.qasm3.load") and [`qiskit.qasm3.loads()`](qasm3#qiskit.qasm3.loads "qiskit.qasm3.loads").
+> *   Support for importing OpenQASM 3 programs and creating [`QuantumCircuit`](/api/qiskit/qiskit.circuit.QuantumCircuit "qiskit.circuit.QuantumCircuit") objects from the input program via two new functions [`qiskit.qasm3.load()`](/api/qiskit/qasm3#qiskit.qasm3.load "qiskit.qasm3.load") and [`qiskit.qasm3.loads()`](/api/qiskit/qasm3#qiskit.qasm3.loads "qiskit.qasm3.loads").
```

Comment:
The link
```markdown
[`QuantumCircuit`](/api/qiskit/qiskit.circuit.QuantumCircuit "qiskit.circuit.QuantumCircuit")
```

Takes me to

```
http://localhost:3000/api/qiskit/qiskit.circuit.QuantumCircuit
```

rather than

```
http://localhost:3000/api/qiskit/0.44/qiskit.circuit.QuantumCircuit
```

Is that the intended behaviour?

Reply:
Thanks for the review! Yes, we want all the release notes to be exactly the same, and with all the links pointing to the current API, regardless of what version of the API docs we are at that moment :+1: